scripts/us_bea/states_gdp/import_industry_data_and_gen_mcf.py

```diff
@@ -0,0 +1,163 @@
+ # Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Pulls data from the US Bureau of Economic Analysis (BEA) on quarterly GDP
+per US state. Saves output as a CSV file.
+
+    Typical usage:
+
+    python3 import_data.py
+"""
+from urllib.request import urlopen
+import io
+import zipfile
+import csv
+import re
+from absl import app
+import pandas as pd
+import import_data
+
+# Suppress annoying pandas DF copy warnings.
+pd.options.mode.chained_assignment = None # default='warn'
+
+class StateGDPIndustryDataLoader(import_data.StateGDPDataLoader):
+    """Pulls GDP industry-level state data from BEA.
+
+    Attributes:
+        df: DataFrame (DF) with the cleaned data.
+    """
+    _STATE_QUARTERLY_INDUSTRY_GDP_FILE = "SQGDP2__ALL_AREAS_2005_2020.csv"
+
+    def download_data(self):
```

Comment:
Could this be shared with the parent class download data? The only difference seems to be the filename, which can be passed in as an argument.